### PR TITLE
Update source.extension.vsixmanifest

### PR DESCRIPTION
**Pull request by julielerman**
added visual studio version14 to the power tools vsixmanifest so this will be discoverable from Visual Studio 2015

### CODE REVIEW

File: src/PowerTools/source.extension.vsixmanifest [hunk 1]
```diff
@@ -31,6 +31,9 @@ When right-clicking on a file containing a derived DbContext class, the followin
       <VisualStudio Version="12.0">
         <Edition>Pro</Edition>
       </VisualStudio>
+      <VisualStudio Version="14.0">
+35	    <Edition>Pro</Edition>
```
Comment (AlbertoMonteiro) [inline src/PowerTools/source.extension.vsixmanifest]: You must remove 35

File: src/PowerTools/source.extension.vsixmanifest [hunk 2]
```diff
@@ -31,6 +31,9 @@ When right-clicking on a file containing a derived DbContext class, the followin
...
+36	  </VisualStudio>
```
Comment (AlbertoMonteiro) [inline src/PowerTools/source.extension.vsixmanifest]: You must remove 36

Comment (ErikEJ) [inline src/PowerTools/source.extension.vsixmanifest]: Use spaces, not tabs

Comment (julielerman) [inline src/PowerTools/source.extension.vsixmanifest]: what the...? I didn't realize those got in there. Must have come over when I copied from the PR I originally submitted to codeplex. Thanks!

Comment (ErikEJ) [inline src/PowerTools/source.extension.vsixmanifest]: @julielerman Any plans to update this PR?

**Comment by ErikEJ** [comment]
@julielerman Any plans to update this PR?

**Comment by julielerman** [comment]
is my edit sufficient? or do I have to start another PR?

**Comment by ErikEJ** [comment]
LGTM - just in time for VS "15" 😄

**Comment by julielerman** [comment]
@ErikEJ I'm sure there will still be a few people using vs2015. I've had over 60,000 hits to that one blog post. There may be a few more still needing it in the future. ;)
@bricelam thanks! As soon as I see it in the gallery I will update my blog post. Now go back to working on the visualizer for EF Core! :)

**Comment by Hotkey** [comment]
We are still waiting for update. When ?

**Comment by ErikEJ** [comment]
@Hotkey In the meantime you can use the download @julielerman prepared in her blog post: http://thedatafarm.com/data-access/installing-ef-power-tools-into-vs2015/

**Comment by Hotkey** [comment]
Yes, sorry I'm saw late ;) Thank you @ErikEJ / PowerTools is the real Power ;)

**Comment by TonyHenrique** [comment]
Can you release this for Visual Studio 2017 ?

**Comment by ErikEJ** [comment]
@TonyHenrique You can download a preview from here: https://github.com/aspnet/EntityFramework6/pull/154

**Pull request merged**